test/integration/support/protractor-config/jasmine-spec-reporter.conf.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
```

Comment:
Cool... wasn't sure if you wanted a repeatable conf for the future [WOOSH SOUND]. The answer is no :) It's gone.